new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java

```diff
@@ -359,6 +439,20 @@ public static V1PodList listPods(String namespace, String labelSelectors) throws
     return v1PodList;
   }
 
+  /**
+   * Copy a directory from Kubernetes pod to local destination path.
+   * @param pod V1Pod object
+   * @param srcPath source directory location
+   * @param destination destination directory path
+   * @throws IOException when copy fails
+   * @throws ApiException when pod interaction fails
+   */
+  public static void copyDirectoryFromPod(V1Pod pod, String srcPath, Path destination)
+      throws IOException, ApiException {
+    Copy copy = new Copy();
```

Comment:
do we need to instantiate a class, can we not implement this as a static method?

Reply:
ok, that's a shame :(